dep_check/main.py

```diff
@@ -206,12 +203,14 @@ def create_graph_use_case(self) -> DrawGraphUC:
 
 def main() -> None:
     try:
-        sys.exit(MainApp().main())
+        MainApp().main()
+    except ForbiddenDepencyError:
+        sys.exit(errno.EPERM)
```

Comment:
I think we'd better use a standard rather than custom return code.

But the real need is to differentiate:
- the nominal case: I found some forbidden import and I return a false return code to the caller.
- a bad argument case: the arguments of the program are illformed.

Reply:
Thanks for the clarification.

https://stackoverflow.com/questions/1101957/are-there-any-standard-exit-status-codes-in-linux

It seems that the 1 & 2 status codes used to describe, respectively, a default error and a command line level error is common.

It may be a good option to consider.

Reply:
ok for me